CheckoutTests/Network/RequestFactoryTests.swift

```diff
@@ -58,7 +58,7 @@ final class RequestFactoryTests: XCTestCase {
       XCTAssertEqual(requestParameters.url, URL(string: "https://www.checkout.com/tokens"))
       XCTAssertEqual(requestParameters.additionalHeaders, [
         "Authorization": "Bearer publicKey",
-        "User-Agent": "checkout-sdk-ios/0.1.0"
+        "User-Agent": "checkout-sdk-ios/4.0.0"
```

Comment:
As a rule of thumb I wouldn't change things that don't need changing. This is a test framework, having a version that clearly stands out may come in handy.